manual/manual/refman/modules.etex

```diff
@@ -198,17 +198,17 @@ The expression @'include' module-expr@ in a structure re-exports in
 the current structure all definitions of the structure denoted by
 @module-expr@.  For instance, if the identifier "S" is bound to the
 module
-\begin{verbatim}
-        struct type t = int  let x = 2 end
-\end{verbatim}
+\begin{caml_example*}{verbatim}
+module S = struct type t = int  let x = 2 end
+\end{caml_example}
 the module expression
```

Comment:
Thank you for responding so quickly !

I added the `module B = ...` because the module expression is not an ocaml declaration itself, so caml_example wouldn't accept it.
Should I leave it as it was or should I reformulate the text to make it match with the code ?

Reply:
Simply reformulate the text seems fine: this is an example for include and using a module definition rather than a structure is still one of the most frequent use case.